en/sql/schema/index_stmt.rst

```diff
@@ -227,6 +240,71 @@ Online unique index while other transactions inserts violates uniqueness
+
+.. note::
+
+    * In general, the number of key values that can be assigned by the system doubles whenever *deduplicate level*\ increases by one. Therefore, the length of the linked list on the overflow page will be halved.
+    * The key value given by the system is obtained through the remainder operation for the page number among the OID information of the record. Therefore, the distribution of key values actually given by the system is determined by the number of pages where records are stored.
+    * The overflow page-linked list is disadvantageous in terms of add, delete, and update performance, but advantageous in terms of retrieval performance. Therefore, it is not desirable to increase the *deduplicate level* to prevent overflow pages from being created. (Adjust the length of the OID overflow page for one key to be within tens or hundreds of pages)
+
+
+There are two ways to specify *deduplicate level* when creating an index.
+
+Implicit method
+
+    This method automatically specifies *deduplicate level* when there is no explicit DEDUPLICATE option specified in the SQL statement. This method is affected by the set value of the system parameter **deduplicate_key_level**.
+    If **deduplicate_key_level** is greater than or equal to 1, *deduplicate level* is automatically set to **deduplicate_key_level**. 
+
+Explicit method
+
+    This is a method in which the user explicitly specifies the **DEDUPLICATE** option in the SQL statement. Regardless of the **deduplicate_key_level** set value, the *deduplicate level* specified by the user is applied.
+    As in the example below, specify the DEDUPLICATE statement directly.
+    
+    .. code-block:: sql
+    
+        CREATE TABLE tbl (a int default 0, b int, c int);
+        CREATE INDEX i_tbl_b on tbl (b) WITH DEDUPLICATE=3 COMMENT 'for deduplicate level 3';
+        CREATE INDEX i_tbl_b_c on tbl (b,c) WITH DEDUPLICATE=7 COMMENT 'for deduplicate level 1';
+
+.. warning::
+
+    * If **deduplicate_key_level** is **\-1**\, it is internally ignored and not applied even if it is specified explicitly. That is, in this case, all indexes are created with *deduplicate level* as **0**.
+    * A column name that begins with "**_dedup_**" cannot be created.
+
+.. note::
+
+    * When creating an index, if the composition of the key field is guaranteed to be UNIQUE, the DEDUPLICATE setting set by the user is ignored and *deduplicate level* is created as **0**.
+        * When the key field includes all key fields constituting a specific Primary Key or Unique Index
+        * However, except when used as an argument of a function
+
+Duplicated index
```

Comment:
```suggestion
Allowing multiple indexes on the same column in the same order
```

"Duplicated" may cause confuse with the term "deduplicate"

Reply:
Also, this section should be promoted to title level 3 Because the contents are not related to the implicit method or explicit method.

Please refer to the following:
```
Chapter 1 Title
===============

Section 1.1 Title
-----------------

Subsection 1.1.1 Title
~~~~~~~~~~~~~~~~~~~~~~
```

Suggestion:
```
Allowing multiple indexes on the same column in the same order
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
OK, thank you.